lib/faker/date.ex

```diff
@@ -35,6 +35,48 @@ if Version.match?(System.version(), ">= 1.3.0") do
     end
 
     @doc """
+    Returns a random date in the past up to today in NaiveDateTime
+    """
+    @spec backward(NaiveDateTime.t) :: NaiveDateTime.t
```

Comment:
Does it seem inconsistent to return `NaiveDateTime` for this function when all other functions in module return `Date`? Would it make more sense to have `Faker.DateTime` and `Faker.NaiveDateTime` modules?